examples/transform_split_docs.js

```diff
@@ -0,0 +1,23 @@
+// example transformation function
+// -- split a row into multiple docs
+var x = function(row) {
+  var doc1 = {
+    _id: row.id,
+    name: row.name,
+    type: 'person'
+  }
+  var doc2 = {
+    _id: row.company_id,
+    name: row.company_name,
+    type: 'company'
+  }
+  for(var i in doc) {
```

Comment:
I'm not sure what this example is doing. It seems to be iterating over a variable "doc" which doesn't appear anywhere else? :)

Reply:
oops sorry that loop makes no sense I’ve no idea what happened :)

Reply:
fixed in 8d6eda9